src/tests/ftest/recovery/pass_1.py

```diff
@@ -0,0 +1,416 @@
+            errors.append(
+                "Checker didn't detect orphan pool! msg = {}".format(query_msg))
+
+        # 7. Disable the checker.
+        dmg_command.check_disable()
+
+        # 8. Try creating a container. It should succeed.
+        cont_create_success = False
+        for _ in range(5):
+            time.sleep(5)
+            try:
+                self.container = self.get_container(pool=self.pool)
+                cont_create_success = True
+                break
+            except TestFail as error:
+                msg = (f"## Container create failed after running checker! "
+                       f"error = {error}")
+                self.log.debug(msg)
+
+        if not cont_create_success:
+            errors.append("Container create failed after running checker!")
+
+        # 9. Show that rdb-pool are recovered. i.e., at least three out of four ranks
+        # should have rdb-pool.
+        hosts = list(set(self.server_managers[0].ranks.values()))
+        count = 0
+        rdb_pool_path = f"/mnt/daos0/{self.pool.uuid.lower()}/rdb-pool"
+        for host in hosts:
+            node = NodeSet(host)
+            if check_file_exists(hosts=node, filename=rdb_pool_path):
+                count += 1
+                self.log.info("rdb-pool found at %s", str(node))
+
+        self.log.info("rdb-pool count = %d", count)
+        if count < 3:
+            errors.append(f"Enough rdb-pool hasn't been recovered! - {count} ranks")
+
+        report_errors(test=self, errors=errors)
+
+    def test_lost_all_rdb(self):
+        """Remove rdb-pool from all mount point from all nodes. Now the pool can’t be
+        recovered, so checker should remove it from both MS and engine.
+
+        1. Create a pool.
+        2. Stop servers.
+        3. Remove /mnt/daos0/<pool_uuid>/rdb-pool from all ranks.
+        4. Start servers.
+        5. Enable and start the checker. It should remove pool from MS and engine.
+        6. Query the checker and verify the message.
+        7. Disable the checker.
+        8. Check that the pool does not appear with dmg pool list.
+        9. Verify that the pool directory was removed from the mount point.
+        10. Un-register the pool so that we don't try to destroy during tearDown.
+
+        Jira ID: DAOS-12067
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,pass_1
+        :avocado: tags=test_lost_all_rdb
+        """
+        # 1. Create a pool.
+        self.pool = self.get_pool()
+
+        # 2. Stop servers.
+        dmg_command = self.get_dmg_command()
+        dmg_command.system_stop()
+
+        # 3. Remove /mnt/daos/<pool_uuid>/rdb-pool from all ranks.
+        hosts = list(set(self.server_managers[0].ranks.values()))
+        nodeset_hosts = NodeSet.fromlist(hosts)
+        command = f"sudo rm /mnt/daos0/{self.pool.uuid.lower()}/rdb-pool"
+        pcmd(hosts=nodeset_hosts, command=command)
```

Comment:
Shouldn't we check the return of `pcmd`? Also, the newer, preferred util is `run_remote`. Example usage:
https://github.com/daos-stack/daos/blob/e9ed12bd487f6b7cfd69637cec2fb519cb8d0aae/src/tests/ftest/launch.py#L2634

Reply:
I didn't find this method, but I updated the code to check the return value.

Reply:
FYI run_remote comes from run_utils